### DISCUSSION TITLE
MobileControl Project

### DISCUSSION THREAD

**Discussion by MrYoksul** [Embarcadero/DelphiFMX4Python#93] [Q&A] [answered]
I cloned the MobileControl project. When I deploy it on a device, some files do not get created in the device. The 'build.zip' file is not present on the device. Why is this file not appearing on the device?

**Comment by lmbelo** [comment, marked as answer]
@MrYoksul you need to add it to the deployment list of your project. You can rather use the [PythonEnvironments](https://github.com/Embarcadero/PythonEnvironments) component for an automatic setup.
